src/python_testing/TC_I_2_4.py

```diff
@@ -0,0 +1,261 @@
+#
```

Comment:
https://github.com/project-chip/connectedhomeip/pull/38300 should be merged before this PR is merged

Reply:
The PR has merged, resolving this.